manifests/ca.pp

```diff
@@ -62,21 +62,45 @@
     mode   => '0750'
   })
 
-  file { "${etc_directory}/openvpn/${name}/easy-rsa" :
-    ensure             => directory,
-    recurse            => true,
-    links              => 'follow',
-    source_permissions => 'use',
-    group              => 0,
-    source             => "file:${openvpn::easyrsa_source}",
-    require            => File["${etc_directory}/openvpn/${name}"],
-  }
-
-  file { "${etc_directory}/openvpn/${name}/easy-rsa/revoked":
+  file { "${etc_directory}/openvpn/${name}/easy-rsa":
     ensure  => directory,
-    mode    => '0750',
     recurse => true,
-    require => File["${etc_directory}/openvpn/${name}/easy-rsa"],
+    links   => 'follow',
+    ignore  => '*.cnf',
+    owner   => 0,
+    group   => 0,
+    mode    => '0755',
+    source  => "file:${openvpn::easyrsa_source}",
+    require => File["${etc_directory}/openvpn/${name}"],
+  }
+
+  exec { "copy *.cnf files from easyrsa source to ${name}":
```

Comment:
I don't think that copying data with an exec resource is a good idea. @alexjfisher any ideas?

Reply:
I'm open for new ideas. From https://github.com/voxpupuli/puppet-openvpn/issues/315 I grab the idea with puppetlabs-rsync which use a exec resource to copy data with rsync.

The current solution is a nasty hack. Thats all what I know.

Reply:
What about restore the old behavior

https://github.com/voxpupuli/puppet-openvpn/commit/9f54ab834654dffb494f3997ca709872d2dedf28#diff-b386e10e7f4de7981ca1d52c90456a0dL127